pkg/env/list.go

```diff
@@ -0,0 +1,30 @@
+package env
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/userio"
+	"github.com/coreeng/developer-platform/pkg/environment"
+)
+
+func Render(env environment.Environment, border bool) string {
```

Comment:
What do you think about rendering it to a table?
Similar to what kubectl does

Reply:
Here is an example:
<img width="846" alt="image" src="https://github.com/coreeng/corectl/assets/26671818/4ec14ecb-b9f1-48b7-bc41-644aa41a791b">

I bet there should be library available for this stuff

Reply:
I like it!
I think it's fine to use the library if it makes our life easier

Reply:
yeah i think this is a great idea